plugins/katello/nightly/user_guide/content_hosts/index.md

```diff
@@ -172,4 +172,22 @@ To set System Purpose attributes for a host, Navigate to the Content Host Detail
 
 ![System Purpose](/plugins/katello/{{ page.version }}/user_guide/content_hosts/system_purpose.png)
 
+## Setting System Purpose attributes via Bulk Actions
 
+Select the Content Hosts that you wish to change the System Purpose attributes on.
+
+Next select the 'Bulk Actions' button in the top right, and choose 'Manage System Purpose'
+
+In the following modal, we can see at the top, how many Content Hosts will be affected by the change. Select the dropdown next to the attributes you want to change, select the desired value and when satisfied with the options, click the 'Assign' button.
```

Comment:
"Modal" is a UI term and users shouldn't be expected to know what it means.
```suggestion
In the following screen, a banner at the top displays how many Content Hosts will be affected by the change. Select the dropdown next to the attributes you want to change, select the desired value and when satisfied with the options, click the 'Assign' button.
```

Reply:
Fixed